client/model-config.json

```diff
@@ -68,5 +68,9 @@
   "InstanceAction": {
     "dataSource": "remote",
     "public": false
+  },
+  "Gateway": {
+    "dataSource": "remote",
+    "public": false
```

Comment:
executors are public, shouldn't this be, too? so they can be directly created by clients (the installers).